### PR TITLE
Squashing some BUGS

### PR DESCRIPTION
**Pull request by rolsanmordua**
- index.html looks a lot more consistent now
- Fixed anchor tag behavior across the site
- Cleaned up resources order a bit 
- Fixed footer icons on hover behavior

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -28,7 +28,11 @@
 			gtag('js', new Date());
 			gtag('config', 'UA-109009244-1');
 		</script>
-
+		<script>
```
Comment (ChiaHuaLu) [inline about.html]: Move this to main.js so that all pages auto implement it. Centralized changing location as well.

File: corporate.html [hunk 2]
```diff
@@ -28,6 +28,11 @@
 			gtag('js', new Date());
 			gtag('config', 'UA-109009244-1');
 		</script>
+		<script>
```
Comment (ChiaHuaLu) [inline corporate.html]: Same. Move to main.js

File: events.html [hunk 3]
```diff
@@ -28,6 +28,11 @@
 			gtag('js', new Date());
 			gtag('config', 'UA-109009244-1');
 		</script>
+		<script>
```
Comment (ChiaHuaLu) [inline events.html]: Move to main.js

File: faq.html [hunk 4]
```diff
@@ -28,6 +28,11 @@
 			gtag('js', new Date());
 			gtag('config', 'UA-109009244-1');
 		</script>
+		<script>
```
Comment (ChiaHuaLu) [inline faq.html]: Same. Move to main.js

File: forms.html [hunk 5]
```diff
@@ -28,6 +28,11 @@
 			gtag('js', new Date());
 			gtag('config', 'UA-109009244-1');
 		</script>
+		<script>
```
Comment (ChiaHuaLu) [inline forms.html]: Same. Move to main.js

File: index.html [hunk 6]
```diff
@@ -28,6 +28,11 @@
 	gtag('js', new Date());
 	gtag('config', 'UA-109009244-1');
 	</script>
+	<script>
```
Comment (ChiaHuaLu) [inline index.html]: Same. Move to main.js

File: resources.html [hunk 7]
```diff
@@ -30,6 +30,11 @@
 
 			gtag('config', 'UA-109009244-1');
 		</script>
+		<script>
```
Comment (ChiaHuaLu) [inline resources.html]: Same. Move to main.js

**Comment by rolsanmordua** [comment]
Valid point, tech debt. Fixed.

File: assets/js/main.js [hunk 8]
```diff
@@ -357,4 +357,10 @@ var settings = {
 
 	});
 
-})(jQuery);
\ No newline at end of file
+})(jQuery);
+
+var shiftWindow = function() { scrollBy(0, -100) };
+window.addEventListener("hashchange", shiftWindow);
+function load() {
```
Comment (ChiaHuaLu) [inline assets/js/main.js]: I don't think this load function does anything... No one I can see is calling it. And when I commented it out, it seemed to still function the same...

File: about.html [hunk 9]
```diff
@@ -159,7 +158,7 @@ <h2 id="our-team">Our Team</h2>
 						<div class="team-cards-row">
 							<div class="team-cards-col">
 								<div class="team-card">
-									<img src="images/officers1819/president.jpg" alt="President of Texas ACM"/>
+									<img src="images/officers1819/president.jpg" onmouseover="this.src='Project-Sciurus.png';" onmouseout="this.src='images/officers1819/president.jpg';" alt="President of Texas ACM" class="pres"/>
```
Comment (ChiaHuaLu) [inline about.html]: Need to rethink how we do this easter egg. Rn, it makes April's card disappear. And it's just So BIG 😉. Kind of obstructs the page and distract from the web page I feel.

Perhaps fade April's image out and swap it with the pres seal without disturbing layout?

File: about.html [hunk 10]
```diff
@@ -317,5 +317,21 @@ <h2 class="officer-name">Santiago <br>Moreno</h2>
 		<!-- Copy to Clipboard and Tooltip functionality for Contact Buttons -->
 		<script src="assets/js/clipboard.min.js"></script>
 		<script src="assets/js/clipboard-tooltips.js"></script>
+
+		<!-- Swap Officer Photo for Seal -->
```
Comment (rolsanmordua) [inline about.html]: Is there anyway to remove the small gap between officer picture and seal when no image is visible to the user? I had previously implemented this but purely through CSS and there was no gap.

Could also consider adding a duration to the seal, especially for mobile. Mobile requires you to tap on another element (i.e. not the officer picture/seal) to revert back to the original image.

Comment (rolsanmordua) [reply about.html]: Furthermore, the fade out away from the officer picture is very nice, however, the seal then just abruptly appears after a small delay with no fade. A fade from picture to picture would look best.

Comment (rolsanmordua) [reply about.html]: Could also consider adding a duration to the seal, especially for mobile. Mobile requires you to tap on another element (i.e. not the officer picture/seal) to revert back to the original image. It's not as straight forward on mobile as it is on desktop to just move mouse away.

**Comment by rolsanmordua** [comment]
Easter egg timing is good. On desktop, the seal fade in behavior can be repeated many times after the initial 3 seconds timing has expired. I tested from N, W, S, E incoming directions and it worked fine. 

However, on mobile, I was only able to replicate seal fade in behavior the first time. Subsequent attempts to reproduce the seal by tapping did not produce the same result. To reproduce the effect I had to refresh the page. Is this intentional? Can this be fixed? This is minor but I believe behavior between desktop and mobile should match.

**Comment by ChiaHuaLu** [comment]
On Mobile, when you click on it, you trigger "on hover". But to trigger it again, you have to click away, then click it. 

I can make on click also call on hover, and that should fix the issue

**Pull request merged**